index.html

```diff
@@ -15,159 +15,204 @@
+                            alt="Github logo"></a></li>
+                <li><a href="angelist.com"><img class="logo-images" src="./assets/icons/Angelist.svg"
+                            alt="Angelist logo"></a></li>
+            </ul>
+        </section>
+    
         <!-- First Page -->
         <section class="work">
             <article class="work-section">
-                <img src="./assets/images/tonic-page-1.png" alt="Snapshot Portfolio">
-
-                <h1 class="project-title-heading">Tonic</h1>
-
-                <article class="frame-2">
-                    <h2 class="client-heading">CANOPY</h2>
-                    <img src="./assets/icons/Counter.png" alt="Counter">
-                    <h2 class="role">Back End Dev</h2>
-                    <img src="./assets/icons/Counter.png" alt="Counter">
-                    <h2 class="year">2015</h2>
+                <img src="./assets/images/tonic-page-1.png" alt="Snapshot Portfolio" class="portfolio-img-1-mobile">
+                <img src="./assets/images/snaposhot-1-desktop-p.png" alt="Snapshot Portfolio"
+                    class="portfolio-img-1-desktop">
+                <article class="work-section-texts">
+                    <h1 class="project-title-heading">Tonic</h1>
+
+                    <article class="frame-2">
+                        <h2 class="client-heading text-mobile">CANOPY</h2>
+                        <h2 class="client-heading text-desktop">CANOPY</h2>
+                        <img src="./assets/icons/Counter.png" alt="Counter">
+                        <h2 class="role text-mobile">Back End Dev</h2>
+                        <h2 class="role text-desktop">Back End Dev</h2>
+                        <img src="./assets/icons/Counter.png" alt="Counter">
+                        <h2 class="year text-mobile">2015</h2>
+                        <h2 class="year text-desktop">2015</h2>
+                    </article>
+
+                    <p class="project-info-p text-mobile">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <p class="project-info-p text-desktop">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+
+                    <ul class="logo-tags">
+                        <li><a href="#"><img src="./assets/icons/Htmllogo.png" alt="Htmllogo"></a></li>
+                        <li><a href="#"><img src="./assets/icons/csslogo.png" alt="csslogo"></a></li>
+                        <li><a href="#"><img src="./assets/icons/jslogo.png" alt="jslogo"></a></li>
+                    </ul>
+                    <div class="button-container">
+                        <button class="button-see-project" type="button">
+                            See project
+                        </button>
```

Comment:
Your current card design is not following the Figma template of your choosing. The width of the card is too much and there is a lot of white space around the card. There is also uneven spacing on the top and bottom of the card. This probably happened while you were trying to implement the changes required by the previous reviewer. Kindly fix it. 
This is how it is looking now.
![image](https://user-images.githubusercontent.com/33268927/171881142-0fcecfd8-89cc-4ee3-9278-951cb33e58f8.png)